mm2src/coins/qrc20/qrc20_tests.rs

```diff
@@ -964,9 +975,14 @@ fn test_validate_maker_payment_malicious() {
         .wait()
         .err()
         .expect("'erc20Payment' was called from another swap contract, expected an error")
-        .to_string();
+        .into_inner();
     log!("error: {}", error);
-    assert!(error.contains("Unexpected amount 1000 in 'Transfer' event, expected 100000000"));
+
+    println!("{:?}", error);
```

Comment:
Let's use `log` macro instead.

Reply:
This is actually just a debug leftover, removed